File: FirmSimulator/Models/Score.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace FirmSimulator.Models
{
    public class Score
    {
        [Key]
        public int ScoreId { get; set; }

        [Required]
        public DateTime Date { get; set; }

        [NotMapped]
        public DateTime StartTime { get; set; }

        [Required]
        public string SettingsDescription { get; set; }

        public string Duration { get; set; }

        public double TotalProfit { get; set; }

        public double ProfitMaximization { get; set; }

        [ForeignKey("User")]
        public string UserEmail { get; set; }
    }
}

File: FirmSimulator/wwwroot/app/scores/score.ts
import { UserViewModel } from "../users/user-view-models";

export class Score {
    scoreId: number;
    date: Date;
    startTime: Date;
    settingsDescription: string;
    duration: string;
    totalProfit: number;
    profitMaximization: number;

    userEmail: string;
}